SECURITY.md

```diff
@@ -1,9 +1,44 @@
-# Reporting security issues
+# Security Policy
 
-The Moby maintainers take security seriously. If you discover a security issue, please bring it to their attention right away!
+The maintainers of this project take security seriously. If you discover
+a security issue, bring it to their attention right away!
 
-### Reporting a Vulnerability
+## Reporting a Vulnerability
 
-Please **DO NOT** file a public issue, instead send your report privately to security@docker.com.
+Please **DO NOT** file a public issue, instead send your report privately
+to [security@docker.com](mailto:security@docker.com).
+
+Reporter(s) can expect a response within within 1-2 business days,
+acknowledging the issue was received. 
+
+## Review Process
+
+After receiving the report, an initial triage and technical analysis is
+performed to confirm the report and determine its scope. We may request
+additional information in this stage of the process.
+
+Once a reviewer has confirmed the relevance of the report, a draft security
+advisory will be created on Github. The draft advisory will be used to discuss
+the issue with maintainers, the reporter(s), and where applicable, other
+affected parties under embargo.
+
+If the vulnerability is accepted, a timeline for developing a patch, public
+disclosure, and patch release will be determined. If there is an embargo
+period on public disclosure before the patch release The reporter(s) are
```

Comment:
Note to self; `The reporter(s)` should be lowercase (or did I drop a snippet here?)

Reply:
updated to lowercase